client/v2/cli/testdata/bank_example.yaml

```diff
@@ -0,0 +1,18 @@
+tx:
+  service: cosmos.bank.v1beta1.Msg
+  rpc_command_options:
+    - rpc_method: Send
+      use: send [from_key_or_address] [to_address] [amount...]
+      positional_args:
+        - proto_field: from_address
+        - proto_field: to_address
+        - proto_field: amount
+          varargs: true
```

Comment:
how would you set flags? I know bank doesn't have any custom flags that aren't set via `AddTxFlagsToCmd`, but I'm just curious what this would look like.

Reply:
I'll add an example